openedx/core/djangoapps/embargo/tests/test_middleware.py

```diff
@@ -82,14 +82,14 @@ def test_non_courseware_url(self):
     @patch.dict(settings.FEATURES, {'EMBARGO': True})
     @ddt.data(
         # request_ip, blacklist, whitelist, is_enabled, allow_access
-        ('173.194.123.35', ['173.194.123.35'], [], True, False),
-        ('173.194.123.35', ['173.194.0.0/16'], [], True, False),
-        ('173.194.123.35', ['127.0.0.0/32', '173.194.0.0/16'], [], True, False),
-        ('173.195.10.20', ['173.194.0.0/16'], [], True, True),
-        ('173.194.123.35', ['173.194.0.0/16'], ['173.194.0.0/16'], True, False),
-        ('173.194.123.35', [], ['173.194.0.0/16'], True, True),
-        ('192.178.2.3', [], ['173.194.0.0/16'], True, True),
-        ('173.194.123.35', ['173.194.123.35'], [], False, True),
+        (u'173.194.123.35', [u'173.194.123.35'], [], True, False),
+        (u'173.194.123.35', [u'173.194.0.0/16'], [], True, False),
+        (u'173.194.123.35', [u'127.0.0.0/32', '173.194.0.0/16'], [], True, False),
```

Comment:
No Unicode indicator for the last string here?  Same question 2 lines below.

Reply:
Good catch! It demonstrates these IP are not required to be unicode. Double checking.

Reply:
It turns out the **u** prefix is not needed for the IP in the second array so I just removed all of them.